/**
 * Exercise 2:   (1) Define an object reference and initialize it to null. Try to call a method
 * through this reference. Now wrap the code in a try-catch clause to catch the exception.
 */
package ErrorHandlingWithExceptions_11;

/**
 * @author Богдан
 *
 */
public class Exercise2 {

    /**
     * @param args
     */
    public static void main(String[] args) {
	try {
	    Test2 obj;
	    
	} catch (Exception e) {
	    
	}

    }

}
class Test2{
    void f(){
	
    }
}
